app/controllers/sessions_controller.rb

```diff
@@ -1,8 +1,10 @@
 class SessionsController < ApplicationController
-    # User enters login data, we assign to a variable and check its params against the database to log them in.  If entered data doesn't match, just reload the login page
+  # User enters login data, we assign to a variable and check its params against the database to log them in.  If entered data doesn't match, just reload the login page
+  skip_before_action :require_login, only: [:new, :create]
+  
 
   def create 
-    @user = User.find_by(username: params[:session][:username])
+    @user = User.find_by(username: params[:session][:username].downcase)
```

Comment:
I don't have an issue with downcasing usernames, but I think we need to do it at account creation as well. Otherwise, they could create an account with an uppercase username but wouldn't be able to log in.

Reply:
Yes, definitely.